packages/bpk-component-banner-alert/src/__snapshots__/AnimateAndFade-test.js.snap

```diff
@@ -57,9 +35,9 @@ exports[`AnimateAndFade should render correctly with animateOnEnter 1`] = `
       "MozTransition": "height 200ms ease ",
       "OTransition": "height 200ms ease ",
       "WebkitTransition": "height 200ms ease ",
-      "height": 0,
```

Comment:
I can't figure out why this has changed - any ideas?

Reply:
It may have something to do with how the underlying mechanism in the AnimateHeight component works, as that does a lot of funky stuff related to height / overflow etc.